codestyle/checkstyle.xml

```diff
@@ -205,9 +205,9 @@
     </module>
 
     <module name="Regexp">
-      <property name="format" value='^((?!throw|=).)* Throwables.propagate\('/>
+      <property name="format" value='^((?!throw).)* Throwables.propagate\('/>
```

Comment:
@kamaci could you please explain this regex? Why not just `Throwables.propagate` without any lookbehind?

Reply:
Yes, no need for lookbehind anymore. I've made the change and committed.